.github/workflows/renovate.yml

```
@@ -0,0 +1,18 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  renovate-config-validator:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)